apps/src/applab/applab.js

```diff
@@ -675,6 +617,15 @@ Applab.init = function (config) {
     studioApp.loadAudio(skin.failureSound, 'failure');
   };
 
+  config.shareWarningInfo = {
+    hasDataAPIs: function () {
+      return Applab.hasDataStoreAPIs(Applab.getCode());
+    },
+    onWarningsComplete: function () {
+      window.setTimeout(Applab.runButtonClick.bind(studioApp), 0);
```

Comment:
good to know. I used the to call globals as `::CreateWindow` in the C++ world just to be explicit about the context, so I assume that habit carried over. In this case, I think I just moved the code from somewhere else, but it may have been my code originally :)